python/lsst/sims/maf/utils/opsimUtils.py

```diff
@@ -5,10 +5,21 @@
 import numpy as np
 from .outputUtils import printDict
 
-__all__ = ['connectOpsimDb', 'writeConfigs', 'createSQLWhere',
+__all__ = ['opsimColMapDict', 'connectOpsimDb', 'writeConfigs', 'createSQLWhere',
            'getFieldData', 'getSimData', 'scaleBenchmarks', 'calcCoaddedDepth']
 
 
+def opsimColMapDict():
+    """Return a dict that maps column names.
```

Comment:
Please add some additional comments documenting why this colMapDict object is available .. this will help people understand when/if they should use the opsimColMapDict and avoid overloading it in the future or asking why particular columns *aren't* set when they want to use them, etc.